remoulade/middleware/middleware.py

```diff
@@ -39,59 +38,61 @@ class Middleware:
     default_before = None  # type: Type[Middleware]
     default_after = None  # type: Type[Middleware]
 
+    def get_option_value(self, option_name, message=None, actor=None, default_value=None):
```

Comment:
I think we could always provide the message and the broker (instead of the actor), that would give us a signature like this:
```
def get_option_value(self, broker, message, option, default=None):
```
Add type hint as well

Then, you can use a return early structure that is often more readable
```
# get option at message level
option = message.options.get(option_name)
if option is not None:
    return option

# get option at actor level
actor = broker.get_actor(message.actor_name)
option =  actor.options.get(option_name)
if option is not None
   return option

# get option at middleware level or return default value
return getattr(self, option_name, default)
```
What do you think ?

Reply:
It's better and is adapted to all the uses of options so far.